pkg/server/server.go

```diff
@@ -296,6 +297,7 @@ func (s *Server) newSVIDRotator(ctx context.Context, serverCA ca.ServerCA, metri
 		Log:         s.config.Log.WithField(telemetry.SubsystemName, telemetry.SVIDRotator),
 		Metrics:     metrics,
 		TrustDomain: s.config.TrustDomain,
+		KeyType:     keymanagerv0.KeyType(s.config.CAKeyType),
```

Comment:
This type conversion isn't necessary if we use the same type (see my note above on the GenerateSigner function).